src/redux/store.ts

```diff
@@ -1,5 +1,6 @@
 import { Action, configureStore, ThunkAction } from '@reduxjs/toolkit'
```

Comment:
We can use `pastelModal` I think instead of `errorModal`. In fact, `pastelModal` was refactored from `errorModal` to be generic modal.